### PR TITLE
Documentation for some commands

### PR DESCRIPTION
**Pull request by Mwthorn**
Filling out TODO documentations

### CODE REVIEW

**Comment by Fortifier42** [comment]
Should fix your conflicts. Also maybe merge into one commit? :)

**Comment by Morphan1** [comment]
@Fortifier42 shhh let it happen

File: src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java [hunk 1]
```diff
@@ -2837,7 +2837,18 @@ public void registerCoreMembers() {
         // @Group core
         //
         // @Description
-        // TODO: Document Command Details
+        // The random command uses Java's built-in random utility to
```
Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java]: Implementation is irrelevant.

File: src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java [hunk 2]
```diff
@@ -2857,6 +2868,16 @@ public void registerCoreMembers() {
         //   - narrate "hello"
         //   - narrate "hey"
         //   }
```
Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java]: Can you delete the old samples or update them while you're at it

File: src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java [hunk 3]
```diff
@@ -2837,7 +2837,18 @@ public void registerCoreMembers() {
...
+        //
+        // Specifying a number as argument will get the next following
+        // scripts commands in the queue to be picked at random.
+        // Example "- random 3" will get the next 3 commands
+        // in the current queue and pick one of those 3 to run,
+        // the other 2 commands, that was in range, will be skipped.
+        //
+        // If braces are used for argument, then it is for every
+        // denizen command, in the brace, one of them will be picked
+        // and the rest, in the brace, will be skipped.
```
Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java]: We use `:` notation not braces now...

Comment (BlackCoyote) [inline src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java]: Are you absolutely positive the colon notation works on the random command?

Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java]: prolly

Comment (Fortifier42) [inline src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java]: It does :3

Comment (BlackCoyote) [inline src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java]: it doesn't

Comment (BlackCoyote) [inline src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java]: the random command requires atleast one argument, a number or the braces,  the colon doesn't count as one, command doesn't work

**Comment by Morphan1** [comment]
ff20c3a

**Pull request closed without merging**